v3/how-to-guides/02-pallet-design/a-add-contracts-pallet/index.mdx

```diff
@@ -56,24 +87,21 @@ Implement the Contracts pallet [configuration traits][contracts-config-rustdocs]
 
    ```rust
    /* --snip-- */
+   Contracts: pallet_contracts,
```

Comment:
This was just patched based on #1042 , perhaps I missed why this is relevant to parameter types

Reply:
Yeah this snippet doesn't look like it belongs here.